solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/inbound/JCSMPInboundChannelAdapter.java

```diff
@@ -168,6 +170,11 @@ protected void doStart() {
 		}
 	}
 
+	private ExecutorService buildThreadPool(int concurrency, String topic) {
+		ThreadFactory threadFactory = new CustomizableThreadFactory("binder-" + topic);
```

Comment:
* Can you change `"binder-"` to `"solace-scst-consumer-"`.
* Instead of using `topic`, can you use the binding name instead? i.e. `consumerProperties.getBindingName()`.

Reply:
done